docs/development/submitting-patches.rst

```diff
@@ -20,8 +20,9 @@ directions on the :doc:`security page </security>`.
 Code
 ----
 
-When in doubt, refer to :pep:`8` for Python code. You can check if your code
-meets our automated requirements by running ``make lint`` against it.
+When in doubt, refer to `the Black Code Style`_ for Python code. You can
+check if your code meets our automated requirements by running ``make lint``
+against it.
```

Comment:
Might want to note here that a developer can also run `make reformat` to have black resolve the Python code formatting errors! The output of `make lint` might also require manual work for some violations (such as unused imports).